UITests/LandscapeToolbarTests.swift

```diff
@@ -0,0 +1,89 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import WebKit
+import UIKit
+
+class LandscapeToolbarTests: KIFTestCase, UITextFieldDelegate {
+    private var webRoot: String!
+
+    override func setUp() {
+        webRoot = SimplePageServer.start()
+    }
+
+    /**
+     * Tests landscape page navigation enablement with the URL bar with tab switching.
+     */
+    func testNavigationWithTabSwitch() {
+        // Rotate to landscape.
+        let value = UIInterfaceOrientation.LandscapeLeft.rawValue
+        UIDevice.currentDevice().setValue(value, forKey: "orientation")
```

Comment:
Try running this test with all of our other UI tests. Does running this test cause all following tests to be stuck in landscape orientation? If so, we should make sure to reset the orientation once the test finishes.